tools/download_model_artifacts.py

```
@@ -11,6 +11,7 @@
 def download_model_artifacts(
     application_name: str,
     artifacts_bucket: str,
+    github_actions_check: bool,  # noqa: FBT001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `FBT001`)

Remove unused `noqa` directive

(RUF100)